### ISSUE TITLE
truncated back propagation through time on large LSTM NNet

### ISSUE THREAD

**Issue by AndresZal** [tflearn/tflearn#726]
Hi, i've been working with tflearn for a couple of days and found it incredibly useful, so kudos to the dev(s). I'm interested in RNNs, but I have long series of 4 features over ~2000 timesteps, so the training is quite slow even on the gpu. Is there any way to use truncated back propagation through time to make the training a little faster?
Or maybe the time series are just to long and I have to kill some steps. 
Thanks!

Andres.

Sample code
` net = tflearn.input_data(shape=[None, 2000, 4])
  net = tflearn.lstm(net, 4, return_seq=True)
  net = tflearn.lstm(net, 4)
  net = tflearn.dropout(net, 0.8)
  net = tflearn.fully_connected(net, 2, activation='softmax')
  net = tflearn.regression(net, optimizer='adam', learning_rate=0.001,
                           loss='categorical_crossentropy')

  # Training
  model = tflearn.DNN(net, tensorboard_verbose=0)
  model.fit(trainX, trainY, validation_set= 0.2,n_epoch=2000, show_metric=True,
            batch_size=128, shuffle=True)`
